typescript/packages/well-log-viewer/src/utils/deepcopy.ts

```diff
@@ -1,3 +1,26 @@
+import type { Template } from "../components/WellLogTemplateTypes";
+
 export function deepCopy<T>(source: T): T {
     return JSON.parse(JSON.stringify(source)) as T;
 }
+
+export function deepCopyTemplate(source: Template): Template {
```

Comment:
This function is more than a deep copy, so maybe it needs a more descriptive name and perhaps a description.

Reply:
It is exactly deep copy but with addition of specific functions which are not copied over (ever, I researched several methods), I will add a comment. How do you suggest I change the name so that it will be more clear?

Reply:
Ok, got it